src/firecracker/src/main.rs

```diff
@@ -99,9 +102,10 @@ fn main() -> ExitCode {
 }
 
 fn main_exec() -> Result<(), MainError> {
-    LOGGER
-        .configure(Some(DEFAULT_INSTANCE_ID.to_string()))
-        .expect("Failed to register logger");
+    // Initialize the logger.
+    log::set_logger(&LOGGER).map_err(MainError::SetLogger)?;
+    log::set_max_level(vmm::logger::DEFAULT_LEVEL);
+    info!("Running Firecracker v{FIRECRACKER_VERSION}");
```

Comment:
Can we move this after some args processing? For example after line 289?

Reply:
That avoid the double printing message of `--version`

Reply:
I would prefer leaving it at the start, its simplest to have it as a startup message.

Reply:
I have a commit down line that removes this startup message https://github.com/JonathanWoollett-Light/firecracker/commit/485cbc3715aedc41fe65231f3bfc188cda915ed1 that would remove this duplication.

Reply:
Added a commit that refactors it like we discussed offline https://github.com/firecracker-microvm/firecracker/pull/4047/commits/c75d6fb573e40a7fd573f5227db06de7044fba5e

Reply:
Looks good! Thanks!